apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md

```diff
@@ -6,73 +6,65 @@ hasSubPageNav: true
+### What is the likelihood of XP customers needing CDP on top of XM + Personalize because without CDP they won&#39;t be able to access customer data as they do in xDB within XP? Can customers personalize on historic data if they don&#39;t have CDP?
 
 Personalize can track users over time and then personalize off of previous visits/history - just like XP. Think of the full CDP as the Data Lake that can be used to aggregate customer data from across the enterprise. Personalize tracks how customers are interacting with testing and experiences -an organization might have additional data that they want to add to a customer&#39;s profile - credit score, payment preferences, demographic data. If an org wants to add that data to the profile so they can use that to drive personalization they would need to upgrade to full CDP.
 
-**How do we reconcile the typical ~9mo sales cycle for Personalize into a \&lt;5mo prospect-\&gt;contract cycle in FY22?** **e.g.** **significant blockers/bottlenecks have been identified and removed?**
+### Can I migrate my XP personalization rules to Sitecore Personalize?
 
-Think of Sitecore Personalize like Adobe Target, Oracle Maximizer, Optimizely, Dynamic Yield, etc. In that world, deal cycles are much faster than CDP and Personalize together which is the more traditional Boxever 9-month cycle.
+You will have to manually recreate your rule logic in Personalize using Decisioning and add it to an experience. Decision Models combined with Web Experiences or Full Stack Interactive Experiences will provide your best equivalent to the power within XP.
 
-**Can I migrate my XP personalization rules to Sitecore Personalize?**
-
-**Current**
-
-No, you will have to manually recreate your rule logic in Personalize using Decisioning and add it to an experience.
-**Future - Coming Soon, not currently Available**
-Experience Targeting: We want the marketer persona to be able to set up an experience that delivers different content based on a pre-defined audience. This pre-defined audience is built upon the same structure as XM/P conditions. The user can combine these conditions into an audience and decide on what output the variant will return. We have high level plans to build some level of integration between XM and Personalize so that the user could set up personalization in XM and it automatically creates an experience with the required variants and assigned audiences. There are some rules in XM that won&#39;t be transferrable. More work to be done here.
-
-**We currently use XM Managed Cloud but like the sound of adding Personalize and Send. We use .Net to publish to the web apps in Azure, will we continue to do that if we move to your new XP2XMP offering?**
+### We currently use XM Managed Cloud but like the sound of adding Personalize and Send. We use .Net to publish to the web apps in Azure, will we continue to do that if we move to your new XP2XMP offering?

```

Comment:
There are references to Send in weird places, I'd suggest but not required, removing all references to Send, since this is purely about XM + Personalize.

Reply:
Good point, removing the references to Send in all the pages (except for Features). My guess is that the original packaging of the offer was to give Personalize and Send to replace XP, but that's not the context here anymore.